tests/test.py

```diff
@@ -2299,6 +2299,65 @@ def testReadAcrastructInAcrastruct(self):
         self.assertEqual(row['empty'], b'')
 
 
+class KMSAWSType:
+    def setUp(self):
+        if not TEST_WITH_AWS_KMS:
+            self.skipTest("test with AWS KMS ACRA_MASTER_KEY loader")
+
+        configuration = {
+            'access_key_id': 'access_key_id',
+            'secret_access_key': 'secret_key_id',
+            'region': os.environ.get('KMS_REGION', 'eu-west-2'),
+            'endpoint':  os.environ.get('AWS_KMS_ADDRESS', 'http://localhost:8080')
+        }
+        self.config_file = tempfile.NamedTemporaryFile('w+', encoding='utf-8')
+        json.dump(configuration, self.config_file)
+        self.config_file.flush()
+        super().setUp()
+
+    def get_kms_type(self):
+        return 'aws'
+
+    def get_kms_configuration_path(self):
+        return self.config_file.name
+
+
+class KMSEncryptorMixing:
+    ZONES = []
```

Comment:
will be great to add comment why we introduce here new list with zones

Reply:
Sure